camel-route-support/src/test/resources/io/kaoto/backend/api/service/step/parser/camelroute/route6-un-marshal.yaml

```diff
@@ -0,0 +1,12 @@
+- from:
+    uri: timer:null
+    steps:
+    - marshal:
+        data-format:
+          json:
+            library: Gson
+    - unmarshal:
+        data-format:
+          json:
+            property: something
+            another: whatever
```

Comment:
it might be worthy to add the other ways to write it without the `data-format` attribute

Reply:
Those are covered on the evil eip, but I will add them here too.

Reply:
Done!